plugins/grid-bookmark/src/GridBookmarkWidget/model.ts

```diff
@@ -159,64 +146,59 @@ export default function f(_pluginManager: PluginManager) {
         bookmark: IExtendedLabeledRegionModel,
         label: string,
       ) {
-        const target = self.bookmarkedRegions.find(
```

Comment:
seemed to me that it was unneeded to find this bookmark in the list, could directly call setLabel on correspondingObj